requirements.txt

```diff
@@ -17,6 +17,7 @@ pylint
 pyright==1.1.280
 pytest
 pywin32 ; sys_platform=="win32"
+requests
```

Comment:
We also need to declare this dependency in `dependencies.ts` (the file chainner uses to manage python deps inside the GUI). 

@joeyballentine Should this be a required dep or an optional dep? `requests` a pretty commonly used (and small) library, so I wouldn't mind making it a required dep.

Reply:
I'm fine with it being required